css/index.css

```diff
@@ -69,6 +70,41 @@ p {
 .container {
     width: 800px;
     margin: 0 auto;
+    display:flex;
+    flex-direction:column;
+    justify-content: space-evenly;
+}
+
+header {
+    display: flex;
+    justify-items: center;
+}
+nav{
```

Comment:
Good job using flexbox to layout your nav image and anchors.  You can also use space-between and it will line up the image and the last anchor to the edge of the screen like you see in the design file.